blockchain/blocks/src/header/json.rs

```diff
@@ -131,3 +137,27 @@ where
         .build_and_validate()
         .unwrap())
```

Comment:
Didn't notice on other, but this shouldn't be unwrap if you don't mind changing here also